Example/src/examples/Image.tsx

```diff
@@ -83,7 +82,8 @@ class DataURI extends Component {
           y="5%"
           width="90%"
           height="90%"
-          href={{uri: dataUriExample}}
+          // @ts-expect-error
```

Comment:
Why is it needed? Is the type of `href` in `Image` wrong?

Reply:
@WoLewicki Yes, it's currently incorrect. I will not display the image on web when data is base64 overwise.